src/views/ProjectView.vue

```diff
@@ -17,7 +17,13 @@ const isPopupVisible = ref(false)
 const uniqueDataSessions = ref([])
 const newSessionName = ref('')
 const errorMessage = ref('')
-const dataSessionsUrl = store.state.dataSessionsUrl
+const dataSessionsUrl = store.state.datalabApiBaseUrl + 'datasessions/'
+
+const authHeaders = {
+    'Content-Type': 'application/json',
+    'Accept': 'application/json',
+    'Authorization': 'Token 123456789abcdefg',
```

Comment:
shouldn't this also use the authtoken from the store?

Reply:
done!